examples/kitchen/.kitchen.linux.yml

```diff
@@ -1,37 +0,0 @@
----
```

Comment:
we should not delete the examples. Instead we should migrate to work with the new version